source/utils/deprecation-warning.ts

```diff
@@ -0,0 +1,14 @@
+const alreadyWarned: string[] = [];
+
+export default (message: string) => {
+	if (alreadyWarned.includes(message)) {
+		return;
+	}
+
+	alreadyWarned.push(message);
+
+	// @ts-ignore
```

Comment:
Please include a comment. If the types are missing, then:

```suggestion
	// @ts-ignore Missing types.
```